blog.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Adopt A Dog</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+</head>
+<body>
+    <div class="header">
+        <h1>Seattle's premiere dog adoption agency</h1>  
+
+    </div>
+    <div class="grid-container">
+    <div class="item1">
+        <nav class="site-nav">
+            <a href="index.html">Home</a>
+            <a href="dog.html">/ Dog</a>
+            <a href="blog.html">/ Blog</a>
+        </nav>
+    </div>
+    </div>
+    <div class="container">
+        <h2>Adopt-A-Dog Blog</h2>
```

Comment:
This h2 tag is a little out of place. It looks like it's part of a blog entry

Reply:
Corrected and moved h2 tag before div tag. Thank you.